sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModelField.cs

```diff
@@ -6,7 +6,7 @@
 namespace Azure.AI.FormRecognizer.Training
 {
     /// <summary>
-    /// A field that the model will extract from forms it analyzes.
+    /// Represents a pre-defined key-value pair that a model will extract from forms it analyzes.
```

Comment:
Let's not use the term "key value pair" for these things.  Their "keys" are not unique, and in .NET this has a specific meaning around dictionaries where the keys are unique.

We can say a form field includes a name unique to the submodel, a field label representing the label of the field on the form, and, if a model was trained with training-time labels, an estimated accuracy for recognition of the field.